content/docs/hooks-custom.md

```diff
@@ -211,16 +210,16 @@ function todosReducer(state, action) {
         text: action.text,
         completed: false
       }];
-    // ... other actions ...
+    // ... otras acciones ...
     default:
       return state;
   }
 }
 ```
 
-Reducers are very convenient to test in isolation, and scale to express complex update logic. You can further break them apart into smaller reducers if necessary. However, you might also enjoy the benefits of using React local state, or might not want to install another library.
+Los reductores son muy convenientes para probar en aislamiento y escalar para expresar una lógica de actualización compleja. Puedes separarlos aun más en reductores mas pequeños si es necesario. Sin embargo, puedes también disfrutar los beneficios de usar el estado local de React, o puedes no querer instalar otra librería.
```

Comment:
```suggestion
Los reductores son muy convenientes para probar en aislamiento y escalar para expresar una lógica de actualización compleja. Puedes separarlos aun más en reductores mas pequeños si es necesario. Sin embargo, puedes también disfrutar los beneficios de usar el estado local de React, o puedes no querer instalar otra biblioteca.
```

Reply:
At the beginning of the last sentence, I suggest this:
> Sin embargo, es posible que también te gusten los beneficios de ...